### DISCUSSION TITLE
Protein and drug IDs

### DISCUSSION THREAD

**Discussion by ChatterjeeAyan** [snap-stanford/ogb#365] [General]
Hi,

Is there a way I can retrieve the protein PDB IDs for the protein-protein interaction network dataset, and the drug PubChem IDs for the drug-drug interaction dataset?

**Comment by weihua916** [comment]
See https://github.com/snap-stanford/ogb/issues/261

**Comment by ChatterjeeAyan** [comment]
I have the latest version of OGB installed, but couldn't find the folder mentioned in #261: ${root}/ogbn_proteins/mapping.
Can you help me navigate to this folder?

**Reply by weihua916** [reply]
It is most likely in dataset/ogbn_proteins/mapping from where you have executed your python code.

**Comment by ChatterjeeAyan** [comment]
Got it. The location is: ogb/examples/linkproppred/ppa/dataset/ogbl_ppa/mapping
